bids-apps.tsv

```diff
@@ -4,6 +4,7 @@ fmriprep_1.4.1	docker://poldracklab/fmriprep:1.4.1	--nthreads 8 --n_cpus 8 --mem
 fmriprep_1.4.0	docker://poldracklab/fmriprep:1.4.0	--nthreads 8 --n_cpus 8 --mem_mb 32000 --omp-nthreads 8 --use-plugin ${FMRIPREP_MULTIPROC_YAML} -w /scratch/${USER}/${SLURM_JOB_ID}_${SLURM_ARRAY_TASK_ID} --fs-license-file ${FS_LICENSE_FILE} --resource-monitor --notrack	Regular
 fmriprep_1.3.2	docker://poldracklab/fmriprep:1.3.2	--nthreads 8 --n_cpus 8 --mem_mb 32000 --omp-nthreads 8 --use-plugin ${FMRIPREP_MULTIPROC_YAML} -w /scratch/${USER}/${SLURM_JOB_ID}_${SLURM_ARRAY_TASK_ID} --fs-license-file ${FS_LICENSE_FILE} --resource-monitor --notrack	Regular
 fmriprep_1.3.0.post2	docker://poldracklab/fmriprep:1.3.0.post2	--nthreads 8 --n_cpus 8 --mem_mb 32000 --omp-nthreads 8 --use-plugin ${FMRIPREP_MULTIPROC_YAML} -w /scratch/${USER}/${SLURM_JOB_ID}_${SLURM_ARRAY_TASK_ID} --fs-license-file ${FS_LICENSE_FILE} --resource-monitor --notrack	Regular
+fmriprep_1.2.6-1	docker://poldracklab/fmriprep:1.1.8	--nthreads 8 --n_cpus 8 --mem_mb 32000 --omp-nthreads 8 --use-plugin ${FMRIPREP_MULTIPROC_YAML} -w /scratch/${USER}/${SLURM_JOB_ID}_${SLURM_ARRAY_TASK_ID} --fs-license-file ${FS_LICENSE_FILE} --resource-monitor --notrack	Regular
```

Comment:
can you also change the container to docker://poldracklab/fmriprep:1.2.6-1 ?

Reply:
Sorry, missed that.  Fixed now.